pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -219,7 +225,9 @@ func (o *ResourcesOptions) Run() error {
 
 		//no changes
 		if string(patch.Patch) == "{}" || len(patch.Patch) == 0 {
-			allErrs = append(allErrs, fmt.Errorf("info: %s %q was not changed\n", info.Mapping.Resource, info.Name))
+			if _, err := fmt.Fprintf(o.Out, "%s %q was not changed\n", info.Mapping.Resource, info.Name); err != nil {
+				return err
```

Comment:
returning short circuits modifying other objects, right?

Reply:
this will explicit indicate object not changed when update multiple object